Add section about kubernetes 1.22 and containerd limitation (#11)
epinio/docs: src/installation/system_requirements.md

In addition, extensive requirements for your workload (apps) would add to that.

A default storage class (with annotation `storageclass.kubernetes.io/is-default-class: "true"`) is needed.

> Kubernetes 1.22 is supported but there is an issue when the container runtime is containerd > 1.5.6

Reviewer: ```suggestion
While Kubernetes 1.22 is supported there is an issue when the container runtime is `containerd > 1.5.6`.
```

> because the [pack cli](https://github.com/buildpacks/pack) is putting too much information inside the

Reviewer: ```suggestion
Starting from that version the [pack cli](https://github.com/buildpacks/pack) places too much information into the
```

Reviewer: Ah. Ok. Not this then.

> because the [pack cli](https://github.com/buildpacks/pack) is putting too much information inside the

Reviewer: Redoing the suggestion then. Right now the phrasing is broken with an incomplete sentence due to the other requests applied.
```suggestion
The [pack cli](https://github.com/buildpacks/pack) is placing too much information into the
```

> image layers. Read more here: https://github.com/paketo-buildpacks/full-builder/issues/415

Reviewer: ```suggestion
image layers. The relevant issue is: https://github.com/paketo-buildpacks/full-builder/issues/415
```
